pkgs/by-name/gu/guile-srfi-145/package.nix

```
@@ -0,0 +1,61 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitea,
+  guile,
+  guile-irregex,
+}:
+stdenv.mkDerivation {
+  pname = "guile-srfi-145";
+  version = "unstable-2023-06-04";
```

```suggestion
  version = "0-unstable-2023-06-04";
```

Versions must start with a digit, see https://github.com/NixOS/nixpkgs/tree/master/pkgs#versioning